src/Esfa.Vacancy.Register.Api/App_Start/WebApiConfig.cs

```diff
@@ -8,6 +9,7 @@ public static class WebApiConfig
         public static void Register(HttpConfiguration config)
         {
             // Web API configuration and services
+            GlobalConfiguration.Configuration.Formatters.JsonFormatter.SerializerSettings.Converters.Add(new StrictEnumConverter());
```

Comment:
The SerializerSettings are already being assigned further down in the class. Can you incorporate this change in there? Otherwise I think this line won't make a change.